pkg/server/sandbox_remove.go

```diff
@@ -39,9 +41,9 @@ func (c *criContainerdService) RemovePodSandbox(ctx context.Context, r *runtime.
 
 	sandbox, err := c.getSandbox(r.GetPodSandboxId())
 	if err != nil {
-		return nil, fmt.Errorf("failed to find sandbox %q: %v", r.GetPodSandboxId(), err)
-	}
-	if sandbox == nil {
+		if !metadata.IsNotExistError(err) {
```

Comment:
Currently, `Get` returns error only when the metadata does not exist.
I could change it to `an error occurred when try to find sandbox %q: %v`. Is that better?